Tella/Scenes/Uwazi/Views/Widgets/Common/UwaziEntityTitleView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UwaziEntityTitleView.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/11/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziEntityTitleView: View {
+    var title: String
+    var body: some View {
+        Group {
+            HStack {
+                Text(title)
+                    .font(.custom(Styles.Fonts.regularFontName, size: 14))
+                    .foregroundColor(Color.white)
+                Text("*")
```

Comment:
this should be rendered conditionally only if the component is required. I think in some cases the component does not need this asterisk

Reply:
Ya as it is a initial draft so left it to only concentrate on the core thing. I have handled it.